### PR TITLE
Signhost pullrequest

### PR DESCRIPTION
**Pull request by DoganGunay**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1044 @@
...
+                  "type": "string",
+                  "description": "The reason of cancellation."
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The transaction is succesfully deleted"
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: Could you please correct the spelling "succesfully"?

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1044 @@
...
+            "required": true,
+            "schema": {
+              "type": "string",
+              "format": "binary"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "File succesfully created and added to the transaction"
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: Could you please correct the spelling "succesfully"?

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1044 @@
...
+                "description": "The id of the signer, must be unique within a transaction.\nIf you don't provide an id we will generate one for you.\n"
+              },
+              "Email": {
+                "type": "string",
+                "description": "The e-mail address of the signer",
+                "example": "john.doe@example.com"
+              },
+              "IntroText": {
+                "type": "string",
+                "description": "An intro text to show to the user during the sign proces.\nThis will be shown on the first screen to the signer and supports limitted markdown markup.\nThe following markup is supported:\n- `# Headings`\n- `*Emphasis*` / `_Emphasis_`\n- `**Stong**` / `__Strong__`\n- `1. Ordered` and `- Unordered` lists\n"
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: Can you please correct this spelling "proces" and "Stong"?

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1044 @@
...
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: Can you please correct this spelling "proces", "limitted" and "Stong"?

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,1044 @@
...
+              "Message"
+            ],
+            "properties": {
+              "Name": {
+                "type": "string",
+                "description": "The name of the receiver."
+              },
+              "Email": {
+                "type": "string",
+                "description": "The e-mail address of the reveiver."
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: Can you please correct this spelling "reveiver"?

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,1044 @@
...
+                        202,
+                        203,
+                        301,
+                        302,
+                        303,
+                        401,
+                        402,
+                        403
+                      ],
+                      "description": "* 101 -  Invitation sent\n* 102 -  Received\n* 103 -  Opened\n* 104 -  Reminder sent\n* 105 -  Document opened, Info property contains the file id of the opened document.\n* 201 -  Cancelled\n* 202 -  Rejected\n* 203 -  Signed\n* 301 -  Signed document sent\n* 302 -  Signed document opended\n* 303 -  Signed document downloaded\n* 401 -  Receipt sent\n* 402 -  Receipt opened\n* 403 -  Receipt downloaded\n"
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: Please correct this spelling "opended"

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,1044 @@
...
+              "Message"
+            ],
+            "properties": {
+              "Name": {
+                "type": "string",
+                "description": "The name of the receiver."
+              },
+              "Email": {
+                "type": "string",
+                "description": "The e-mail address of the reveiver."
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: Please correct this spelling "reveiver"

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,1044 @@
...
+                },
+                "Location": {
+                  "type": "object",
+                  "description": "Specify where the field should be placed within the document.",
+                  "properties": {
+                    "Search": {
+                      "type": "string",
+                      "description": "The text to search in the pdf document to use as the position for the field. For example `{{Signer1}}`."
+                    },
+                    "Occurence": {
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: can you please correct this spelling "occurence"?

**Review by vmanoharas — COMMENTED** [review]
Hello @DoganGunay, 

I have completed my first round of review, can you please resolve review comments?

File: certified-connectors/SignHost/apiDefinition.swagger.json (hunk 8 above)
Comment (DoganGunay) [reply certified-connectors/SignHost/apiDefinition.swagger.json]: Removed the property completely as it was unused

File: certified-connectors/SignHost/Readme.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+# SignHost
```
Comment (vmanoharas) [inline certified-connectors/SignHost/Readme.md:1]: Can you please update readme.md file as per the below template?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,945 @@
...
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/Authentication"
+                }
+              },
+              "Verifications": {
+                "description": "List of verifications that the signer has to verify with.\nThe order in which the verifications are provided determine in which order the signer will have to perform the specified method.\nVerifications must be performed before the document(s) can be signed.\nYou **must** use one of the following verifications as the last method:\n- Consent\n- itsme sign<sup>*</sup>\n- PhoneNumber\n- Scribble\n- SigningCertificate<sup>*</sup>\n- ZealiD Qualified<sup>*</sup>\n<sup>*</sup> These verifications can not be used in any other position than the last.\n",
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 11]
```diff
@@ -0,0 +1,945 @@
...
+                        202,
+                        203,
+                        301,
+                        302,
+                        303,
+                        401,
+                        402,
+                        403
+                      ],
+                      "description": "* 101 -  Invitation sent\n* 102 -  Received\n* 103 -  Opened\n* 104 -  Reminder sent\n* 105 -  Document opened, Info property contains the file id of the opened document.\n* 201 -  Cancelled\n* 202 -  Rejected\n* 203 -  Signed\n* 301 -  Signed document sent\n* 302 -  Signed document opened\n* 303 -  Signed document downloaded\n* 401 -  Receipt sent\n* 402 -  Receipt opened\n* 403 -  Receipt downloaded\n"
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 12]
```diff
@@ -0,0 +1,945 @@
...
+                "type": "string",
+                "description": "The e-mail address of the signer",
+                "example": "john.doe@example.com"
+              },
+              "IntroText": {
+                "type": "string",
+                "description": "An intro text to show to the user during the sign process.\nThis will be shown on the first screen to the signer and supports limitted markdown markup.\nThe following markup is supported:\n- `# Headings`\n- `*Emphasis*` / `_Emphasis_`\n- `**Strong**` / `__Strong__`\n- `1. Ordered` and `- Unordered` lists\n"
+              },
+              "Authentications": {
+                "description": "List of authentications that the signer has to authenticate with.\nThe order in which the authentications are provided determine in which order the signer will have to perform the specified method.\nAuthentications must be performed before the document(s) can be viewed.\nYou **must** explicitly specify the API-version when using this feature.\nThis is done with the header: 'Accept: application/vnd.signhost.v1+json'.\n",
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

File: certified-connectors/SignHost/apiDefinition.swagger.json [hunk 13]
```diff
@@ -0,0 +1,945 @@
...
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/Authentication"
+                }
+              },
+              "Verifications": {
+                "description": "List of verifications that the signer has to verify with.\nThe order in which the verifications are provided determine in which order the signer will have to perform the specified method.\nVerifications must be performed before the document(s) can be signed.\nYou **must** use one of the following verifications as the last method:\n- Consent\n- itsme sign<sup>*</sup>\n- PhoneNumber\n- Scribble\n- SigningCertificate<sup>*</sup>\n- ZealiD Qualified<sup>*</sup>\n<sup>*</sup> These verifications can not be used in any other position than the last.\n",
```
Comment (vmanoharas) [inline certified-connectors/SignHost/apiDefinition.swagger.json]: I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).

**Review by vmanoharas — COMMENTED** [review]
Hello @DoganGunay, 

Can you please resolve review comments?

File: certified-connectors/SignHost/apiDefinition.swagger.json (hunk 13 above)
Comment (DoganGunay) [reply certified-connectors/SignHost/apiDefinition.swagger.json]: shortened all necessary descriptions

File: certified-connectors/SignHost/apiDefinition.swagger.json (hunk 12 above)
Comment (DoganGunay) [reply certified-connectors/SignHost/apiDefinition.swagger.json]: Shortened all necessary descriptions

**Review by vmanoharas — APPROVED** [review]
Hello @DoganGunay 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**